src/tool_help.c

```diff
@@ -275,6 +275,8 @@ static const struct helptxt helptext[] = {
    "OAuth 2 Bearer Token"},
   {"-o, --output <file>",
    "Write to file instead of stdout"},
+  {"    --overwrite",
+   "Don't overwrite files which already exist"},
```

Comment:
isn't this supposed to be overwrite files why does it say don't

Reply:
Sure, did a typo

Reply:
Done